### PR TITLE
Updating gh pages

### PR DESCRIPTION
**Pull request by AlexChesters**
As per #292 I felt that that this [page](http://fmtvp.github.io/tal/getting-started/introducing-tal.html) was particularly out-of-date. It had no reference to Red Button+ and the screenshots for iPlayer were very old. 

I've added a new screenshot for iPlayer, as well as added information about Red Button+

### CODE REVIEW

**Comment by jmalk** [comment]
This seems entirely reasonable. Nitpicking; I'd rename iPlayer_new to iPlayer_screenshot because nothing stays new for long. (How philosophical.)

File: getting-started/introducing-tal.md [hunk 1]
```diff
@@ -36,16 +36,26 @@ that it can be deployed to all HTML-based TV devices.
...
       <p class="lead">TAL originated from iPlayer</p>
       <p>TAL was extracted from iPlayer as a standalone library to enable further app development.</p>
     </div>
   </li>
+  </ul>
+
+<ul class="thumbnails">
+  <li class="span8 offset2">
+    <div class="thumbnail">
+      <a href="{{site.baseurl}}/img/getting-started/iPlayer.jpg"><img src="{{site.baseurl}}/img/getting-started/RedButtonPlus.jpg" alt=""></a>
```
Comment (tsadler1988) [inline getting-started/introducing-tal.md]: This link is broken - it links to the old iPlayer image.

File: getting-started/introducing-tal.md [hunk 2]
```diff
@@ -36,16 +36,26 @@ that it can be deployed to all HTML-based TV devices.
...
+      <p class="lead">BBC Red Button+</p>
+      <p>Red Button+ was build using TAL.</p>
```
Comment (tsadler1988) [inline getting-started/introducing-tal.md]: The grammar police are here! 

'built' not 'build'.

File: getting-started/introducing-tal.md [hunk 3]
```diff
@@ -36,16 +36,26 @@ that it can be deployed to all HTML-based TV devices.
...
 </ul>
 
-*iPlayer*, *BBC News*, and *BBC Sport* are three high profile TV applications built on TAL. Click on the images for larger versions:
+*iPlayer*, *Red Button+*, *BBC News*, and *BBC Sport* are four high profile TV applications built on TAL. Click on the images for larger versions:
 
 <ul class="thumbnails">
   <li class="span8 offset2">
     <div class="thumbnail">
-      <a href="{{site.baseurl}}/img/getting-started/iPlayer.jpg"><img src="{{site.baseurl}}/img/getting-started/iPlayer_50.jpg" alt=""></a>
+      <a href="{{site.baseurl}}/img/getting-started/iPlayer.jpg"><img src="{{site.baseurl}}/img/getting-started/iPlayer_new.jpg" alt=""></a>
```
Comment (tsadler1988) [inline getting-started/introducing-tal.md]: This link is broken - it links to the old iPlayer image.

This probably means you want a large new iPlayer screenshot to link to.

**Comment by tsadler1988** [comment]
Thanks for this Alex, these pages get out of date all to easily!

Further to Josh's comment, I would suggest that you can delete iPlayer.jpg and iPlayer_50.jpg. I searched the gh-pages source code and they will no longer be used once this gets merged in.

**Comment by AlexChesters** [comment]
Hi @tsadler1998, @jmalk,

I've fixed the typo Josh mentioned and made some changes to the images

Would one of you mind taking another look at this?

Cheers!

**Comment by tsadler1988** [comment]
If I was going to be super nit-picky I would demand 720p images of iPlayer and RB+ for the page to link to, as is available for Sport and News, but I don't think that should stop this being merged in. 

Perhaps adding 720p images can be a later enhancement.

**Pull request merged**